benchmarks/egg.js

```diff
@@ -6,7 +6,7 @@ const app = new Application()
 app.config.keys = 'my secret keys'
 
 app.router.get('/', ctx => {
-  ctx.body = 'hello world'
+  ctx.body = JSON.stringify({hello:"world"})
```

Comment:
This will negatively impact the benchmarks for this framework due to the non-zero cost of `JSON.stringify`.

Reply:
But in other cases (even in bare version) JSON.stringify is used. 
Therefore, it would be dishonest to give such indulgence only to this framework.

Reply:
If that is the case, then it's fine by me. @fastify/benchmarks ?

Reply:
@fastify/benchmarks ?